docs/usage.md

```diff
@@ -37,12 +37,71 @@ spec:
 3. Or you can pass the config-management-plugin flag to the Argo CD CLI app create command:  
 `argocd app create you-app-name --config-management-plugin argocd-vault-plugin`
 
+#### With `helm`
```

Comment:
Lets not do helm, kustomize or jsonnet with the back ticks, i think it throws off the formatting